In most situations, you can capture errors automatically with `captureException()`.

```javascript
try {
  aFunctionThatMightFail();
} catch (err) {
  Sentry.captureException(err);
}
```
